### PR TITLE
bash-completion: Update port completion

### PR DESCRIPTION
**Pull request by judaew**
Add bump keyword to $cword

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.5 21G72 x86_64
Command Line Tools 13.4.0.0.1.1651278267

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Review by mascguy — APPROVED** [review]
LGTM. @raimue ?

File: sysutils/bash-completion/files/port [hunk 1]
```diff
@@ -30,7 +30,7 @@ _port()
                         mode=cmdfile
                         cmdfile=${words[((count+1))]}
                         ;;
-                    activate|archive|archivefetch|build|cat|cd|checksum|clean|configure|\
+                        activate|archive|archivefetch|build|bump|cat|cd|checksum|clean|configure|\
```
Comment (raimue) [inline sysutils/bash-completion/files/port:33]:
```suggestion
                    activate|archive|archivefetch|build|bump|cat|cd|checksum|clean|configure|\
```

Please keep indentation aligned with the other options.

**Review by raimue — COMMENTED** [review]
Looks good for me except for the suggested whitespace change to keep original indentation.

File: sysutils/bash-completion/files/port (hunk 1 above)
Comment (judaew) [reply sysutils/bash-completion/files/port]: Done

**Pull request merged**